x-pack/packages/kbn-elastic-assistant-common/index.ts

```diff
@@ -25,4 +25,6 @@ export {
 export { transformRawData } from './impl/data_anonymization/transform_raw_data';
 export { handleBedrockChunk } from './impl/utils/bedrock';
 
+export type { Message as LLMMessage } from './impl/llm/types';
```

Comment:
@stephmilovic don't like naming here, but Message type already exists in `schemas`. If you have any thoughts how to improve it, please share